pkg/virt-controller/watch/vm/vm_test.go

```diff
@@ -7010,6 +7010,55 @@ var _ = Describe("VirtualMachine", func() {
 			})
 		})
 
+		Context("feature configurables", func() {
+			It("should update the VM conditions if the downwardMetrics is disabled and it is requested in a stopped VM", func() {
+				vm, _ := watchtesting.DefaultVirtualMachine(false)
+				vm.Spec.Template.Spec.Domain.Devices.DownwardMetrics = &v1.DownwardMetrics{}
+
+				vm, err := virtFakeClient.KubevirtV1().VirtualMachines(vm.Namespace).Create(context.Background(), vm, metav1.CreateOptions{})
+				Expect(err).To(Succeed())
+				addVirtualMachine(vm)
+
+				sanityExecute(vm)
+
+				vm, err = virtFakeClient.KubevirtV1().VirtualMachines(vm.Namespace).Get(context.Background(), vm.Name, metav1.GetOptions{})
+				Expect(err).To(Succeed())
+
+				cond := virtcontroller.NewVirtualMachineConditionManager().GetCondition(vm, v1.VirtualMachineFailure)
+				Expect(cond).ToNot(BeNil())
+				Expect(*cond).To(gstruct.MatchFields(gstruct.IgnoreExtras, gstruct.Fields{
+					"Type":    Equal(v1.VirtualMachineFailure),
+					"Reason":  Equal(virtcontroller.FeatureNotEnabled),
+					"Message": Equal("DownwardMetrics feature is not enabled"),
+					"Status":  Equal(k8sv1.ConditionTrue),
+				}))
+			})
+
+			It("should emit an event when the VM tries to create the VMI object if the downwardMetrics is requested but not enabled", func() {
```

Comment:
This looks like a duplicate of the previous test case. I'd consider merging them.

Reply:
Combined in a DescribeTable.